src/share/util/mct_mod.F90

```diff
@@ -98,6 +98,7 @@ module mct_mod
    use m_GlobalSegMap       ,only: mct_gsMap_gstorage     => GlobalStorage
    use m_GlobalSegMap       ,only: mct_gsMap_ngseg        => ngseg
    use m_GlobalSegMap       ,only: mct_gsMap_nlseg        => nlseg
+   use m_GlobalSegMap       ,only: mct_gsMap_OP           => OrderedPoints
```

Comment:
This is an error.  It requires a minor change in the clm component to remove it, but it should be removed and replaced with line 106.